product_docs/docs/pem/8/tuning_performance/performance_diagnostic.mdx

```diff
@@ -11,7 +11,9 @@ redirects:
 The Performance Diagnostic dashboard analyzes the database performance for Postgres instances by monitoring the wait events. To display the diagnostic graphs, PEM uses the data collected by the EDB Wait States module.
 
 !!! Note
-    For PostgreSQL databases, Performance Diagnostics is supported only for versions 10, 11, and 12 that are installed on the supported CentOS or RHEL platforms.
+    - For PostgreSQL databases, Performance Diagnostics is supported for versions 10 or later installed on the supported CentOS or RHEL platforms.
```

Comment:
I think the word version should be singular.

Reply:
changes done